src/main/java/io/jsonwebtoken/JwtParts.java

```diff
@@ -0,0 +1,72 @@
+package io.jsonwebtoken;
+
+/**
+ * Container class which contains encoded header, payload and signature parts of
+ * the token.
+ *
+ * @since 0.8
+ */
+public class JwtParts {
```

Comment:
Because JWE is being implemented in the `jwe` branch, you might not need to worry about jwe parts yet, but whatever implementation we create should be extensible enough to support that upcoming feature/release.